app/js/controllers/angel.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
+/* global angular, L */
+/* jshint unused:false */
+
+var controllersModule = require('./_index');
+var result;
+var startupslocation=[];
+var marker=[];
+var plotted=0;
+/**
+ * @ngInject
+ */
+
+ controllersModule.controller('AngelCtrl', ['$rootScope','$location','$http','$scope','AppSettings','MapPopUpTemplateService', function($rootScope,$location,$http,$scope,AppSettings,MapPopUpTemplateService) {
+  
+
+
+  var totalpage=0;
+  var currentpage=0;
+  $scope.startups=[];
+  var startupslayer = new L.LayerGroup();
+  var overlays = {
+            "Startups" : startupslayer 
+        };
+  var grayscale=L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+            maxZoom: 18,
+            attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, ' +
+        '<a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
+        'Imagery © <a href="http://mapbox.com">Mapbox</a>' +
+                'Imagery © <a href="http://mapbox.com">Mapbox</a>',
+            id: 'examples.map-20v6611k'
+        });
+        var basemapObj = {
+            "First Basemap": grayscale
+        };
+      
+      var map2 =L.map('map2',{layers:[grayscale,startupslayer]}).setView([20,0], 2);
+      
+      var code = function () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Wrap an immediate function invocation in parentheses.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2550181458/issues/source?bid=3288768&fileBranchId=3288808#l39)